gdal_subsetter/transform.py

```diff
@@ -696,7 +696,8 @@ def recolor(self, layerid, srcfile, dstdir):
                             if relatedUrl.type == 'Color Map':
                                 colormap = '/vsicurl/' + relatedUrl.url
                                 discrete = True
-        if colormap is None and ('png' in fmt.mime or 'jpeg' in fmt.mime):
+        image_types = ['png', 'jpeg', 'tiff']
+        if colormap is None and any(i_type for i_type in image_types if i_type in fmt.mime):
```

Comment:
Could this `any` statement be simplified to:

```python
if colormap is None and any(i_type in fmt.mime for i_type in image_types):
    ...
```

Although this condition feels like it was originally a little off. I _think_ what's trying to be checked here is whether the MIME type is a PNG, JPEG or (now) a TIFF. In which case, it might be better to have:

```python
if colormap is None and fmt.mime in ['image/png', 'image/jpeg', 'image/tiff', 'image/tif']:
    ...
```

Note that `image/tiff` and `image/tif` are both valid MIME types for a TIFF, so that should be included in whatever ends up happening. (I wonder also if it could/should be `fmt.mime.lower()`, to ensure everything is lowercase)

Reply:
Ugh, yeah, that was ugly what I was trying to do.  bbf694d is correct version